DungeonMaster/Assets/Scripts/Models/Skills.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+
+[Serializable]
+public class Skills
+{
+    public static Skills defaultSkills = new Skills(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);
```

Comment:
Would highly suggest implementing the builder pattern for complex constructors, eg  http://www.stefanoricciardi.com/2010/04/14/a-fluent-builder-in-c/

Reply:
https://github.com/oneleif/DungeonMaster-Client/issues/14